uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-entitlements.js

```diff
@@ -1,7 +1,11 @@
+/* eslint-env amd */
+
 define([
+  'common/lodash',
+  'common/moment',
   'leave-absences/absence-tab/modules/components',
-], function (components) {
-
+  'common/models/contract'
+], function (_, moment, components) {
```

Comment:
lodash, moment, contract are not used.
Please remove anything which is not used in a file.